frontends/bmd/public/fonts/noto-emoji.css

```diff
@@ -0,0 +1,41 @@
+/* Sourced from <https://google-webfonts-helper.herokuapp.com/fonts/noto-emoji?subsets=emoji> and then edited to only contain necessary font files. */
+
+/* noto-emoji-300 - emoji */
+@font-face {
+  font-family: 'Noto Emoji';
+  font-weight: 300;
+  font-style: normal;
+  src: local(''), url('noto-emoji-v26-emoji-300.woff2') format('woff2');
+}
+
+/* noto-emoji-regular - emoji */
+@font-face {
+  font-family: 'Noto Emoji';
+  font-weight: 400;
+  font-style: normal;
+  src: local(''), url('noto-emoji-v26-emoji-regular.woff2') format('woff2');
+}
+
+/* noto-emoji-600 - emoji */
+@font-face {
+  font-family: 'Noto Emoji';
+  font-weight: 600;
+  font-style: normal;
+  src: local(''), url('noto-emoji-v26-emoji-600.woff2') format('woff2');
+}
```

Comment:
The most minor nit that you're welcome to ignore: We could move this below `/* noto-emoji-500 - emoji */` such that the font faces are ordered in terms of increasing weight (here and in the other `noto-emoji.css` files)

Reply:
👍  Fixed. I appreciate the nit. This was not on purpose. I used a tool which auto-generated this so that I didn't have to create it by hand. Not sure why the tool ordered them this way.